### DISCUSSION TITLE
Action columns do not display anything

### DISCUSSION THREAD

**Discussion by ArtMin96** [rappasoft/laravel-livewire-tables#435] [Q&A] [answered]
I do not know what I am doing wrong that does not work.

![Syxou36](https://user-images.githubusercontent.com/29732308/129422369-51d5d347-757d-4d5b-bc48-5f0396112c7e.png)

Code:

![w2SSf8r](https://user-images.githubusercontent.com/29732308/129422449-80bb4d3e-7eca-4e4b-af24-b3cf36c13831.png)

![QhndK7Q](https://user-images.githubusercontent.com/29732308/129422504-a693a68d-0615-4c8f-8319-55503cb5e6ee.png)

**Comment by rappasoft** [comment, marked as answer]
If you just return the link from the format closure does it show up?

**Reply by ArtMin96** [reply]
I used this passage -> [https://github.com/rappasoft/laravel-livewire-tables/wiki/Using-the-built-in-cell-formatting#you-could-also-return-a-view](https://github.com/rappasoft/laravel-livewire-tables/wiki/Using-the-built-in-cell-formatting#you-could-also-return-a-view)

**Reply by ArtMin96** [reply]
Fixed. Thank you

**Reply by Todomed** [reply]
How did you solve it? I get the error:
`Undefined variable $row`

**Reply by ArtMin96** [reply]
> How did you solve it? I get the error: `Undefined variable $row`

Class

in columns add this code

```php
Column::make('Actions')
    ->excludeFromSelectable(),
```

add add this

```php
public function rowView(): string
{
    return 'some.path.includes.row';
}
```
